build/source/elmfire_level_set.f90

```diff
@@ -329,45 +328,21 @@ SUBROUTINE LEVEL_SET_PROPAGATION(IWX_BAND,ICASE,NTIMESTEPS)
    IF (R0 .LE. 0.01*SMOKE_OUTPUTS_DUMP_PERCENT) DUMP_SMOKE_OUTPUTS = .TRUE.
 ENDIF
 
+IF (USE_EULERIAN_FRAMEWORK) THEN
```

Comment:
Let's change this variable name to USE_EULERIAN_SPOTTING instead of USE_EULERIAN_FRAMEWORK (since the spread model always uses an Eulerian framework!)